source/slang/slang-ir.h

```diff
@@ -2072,6 +2080,17 @@ struct IRModule : RefObject
 
     IRDeduplicationContext* getDeduplicationContext() const { return &m_deduplicationContext; }
 
+    IRDominatorTree* findDominatorTree(IRGlobalValueWithCode* func)
+    {
+        IRAnalysis* analysis = m_mapInstToAnalysis.tryGetValue(func);
+        if (analysis)
+            return analysis->getDominatorTree();
+        return nullptr;
+    }
+    IRDominatorTree* findOrCreateDominatorTree(IRGlobalValueWithCode* func);
+    void invalidateDominatorTree(IRGlobalValueWithCode* func) { m_mapInstToAnalysis.remove(func); }
```

Comment:
Yeah, it seems like this one could be `invalidateAnalysisForInst(...)`.

Reply:
Done.